apps/backoffice-v2/src/domains/alerts/fetchers.ts

```diff
@@ -103,3 +149,23 @@ export const assignAlertsByIds = async ({
 
   return handleZodError(error, alerts);
 };
+
+export const updateAlertsDecisionByIds = async ({
+  decision,
+  alertIds,
+}: {
+  decision: TAlertState;
+  alertIds: string[];
+}) => {
+  const [alerts, error] = await apiClient({
+    url: `${getOriginUrl(env.VITE_API_URL)}/api/v1/external/alerts/decision`,
+    method: Method.PATCH,
+    body: {
+      decision,
+      alertIds,
+    },
+    schema: z.any(),
```

Comment:
why `z.any()`?

Reply:
The schema is still changing a lot and often. I think its best to address the schemas before merging to dev.

Reply:
add a todo

Reply:
I think that instead of a TODO, you should open a Linear task with a `Tech Debt` label

Reply:
Was added as a task.